package.json

```diff
@@ -84,7 +84,7 @@
     "csrf": "3.1.x",
     "express": "4.17.x",
     "google-libphonenumber": "3.2.6",
-    "govuk-frontend": "3.3.0",
+    "govuk-frontend": "^3.4.0",
```

Comment:
I think we're moving away from using wildcard version in package json in favour of depndabot updating these to fixed version. 

It means `npm install` will always give you the same package in all environments and won't ever try to update things.

Reply:
Ah ok will amend, I had ran `npm i -S govuk-frontend@latest` guess I should have used `npm i -S govuk-frontend@3.4.0`